tests/test_utils.py

```diff
@@ -0,0 +1,21 @@
+class MockResponse:
+    def __init__(self, json_content):
+        self.json_content = json_content
+
+    def json(self):
+        return self.json_content
+
+
+def mock_get_factory(target_url, json_content):
+    """generate a mock to patch request.get with a json response"""
+
+    def mock_get(url, **kwargs):
+        assert url == target_url
+        assert kwargs == {
+            "fields": "product_name,code,last_editor,last_edit_dates_tags",
+            "sort_by": "last_modified_t",
+            "labels_tags_en": "vegan",
+        }
```

Comment:
```suggestion
def mock_get_factory(target_url, expected_kwargs={}, json_content=None):
    """generate a mock to patch request.get with a json response"""

    def mock_get(url, **kwargs):
        assert url == target_url
        assert kwargs == expected_kwargs
```

**Note**: beware I changed the function signature, so the case where you don't expect kwargs you should use named arguments for json_content:
```python
mock_get_factory(target_url, json_content=json_content)
```